go.mod

```diff
@@ -4,7 +4,8 @@ go 1.12
 
 require (
 	github.com/ipfs/go-block-format v0.0.2
-	github.com/ipfs/go-cid v0.0.6
+	github.com/ipfs/go-cid v0.0.7
+	github.com/ipfs/go-ipfs-blockstore v1.0.1
```

Comment:
This would be a really unfortunate dependency for this and, transitively, actors to have. And look at all the stuff it brings in! Happily, I can see nowhere where you've actually imported this so I think you can remove it.

If we do need some testing or benchmarking thing that needs large on-disk storage like this, I think we should pull it out to a separate repo that mere users of the AMT can avoid depending on. 

Blocking, if you can't remove it.

Reply:
I'm not sure how I ended up pulling that in. Go mod tidy took care of it.